docling/backend/genos_hwp_backend.py

```
@@ -475,6 +508,59 @@ def is_really_image(file_path):
+    def _decode_latex_b64(raw: str) -> Optional[str]:
+        """SDK가 emit하는 base64 인코딩된 LaTeX 문자열을 디코드한다.
+        긴 값은 SDK가 중간에 줄바꿈/공백을 끼울 수 있어 전처리 후 디코드한다.
+        실패 시 None 반환.
+        """
+        if not raw:
+            return None
+        # 공백/줄바꿈/HTML 잔여물 제거
+        cleaned = re.sub(r"\s+", "", raw)
+        try:
+            decoded = base64.b64decode(cleaned, validate=False).decode("utf-8", errors="replace").strip()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)